File: src/LPII/Atividade2/Exercicio7.java
package LPII.Atividade2;
import java.util.Scanner;

public class Exercicio7 {
    public static void main(String[] args) {
        Scanner ler = new Scanner(System.in);

        int[] vet = new int[10];
        int i, num;

        System.out.println("Voce vai digitar 10 numeros inteiros");

        for(i=0; i<10; i++){
            System.out.println("Digite um numero");
            vet[i] = ler.nextInt();
        }

        System.out.println("\nDigite um numero para conferencia");
        num = ler.nextInt();

        for(i=0; i<10; i++)
            if(num==vet[i])
                System.out.println("\nO valor digitado esta no vetor\n Valor = " +vet[i]);
            else if(num!=vet[i])
                System.out.println("\nO valor digitado nao esta no vetor\n Valor = " +vet[i]);
    }
}

File: src/LPII/Atividade1/Exercicio7.c
package LPII.Atividade1;

public class Exercicio7 {
    #include <stdio.h>
    #define v 10

    int main(){

        int vet[v], cont, num;

        printf("Voce vai digitar 10 numeros inteiros\n\n");

        for(cont=0; cont<v; cont++){
            printf("Digite o %d numero\n", cont+1);
            scanf("%d", &vet[cont]);
        }
        printf("\nDigite um numero para conferencia\n");
        scanf("%d", &num);

        printf("\n\n");
        for(cont=0; cont<v; cont++)
            if(num==vet[cont])
                printf("O valor %d esta no vetor %d\n",vet[cont], cont+1);
            else if(num!=vet[cont])
                printf("O valor %d nao esta no vetor\n", vet[cont]);

        return 0;
    }

}
